src/TestExplorer/TestExplorer.ts

```diff
@@ -205,12 +231,12 @@ export class TestExplorer {
                             classItem.children.forEach(funcItem => {
                                 const testName = `${targetItem.label}.${classItem.label}/${funcItem.label}`;
                                 if (
-                                    !results.find(item => item === testName) &&
+                                    !results.find(item => item === testName) /* &&
```

Comment:
Do we want to get rid of this?

Reply:
Both of these will be deleted in the next PR which creates a test explorer update from `swift test list`.